packages/amplication-client/src/Entity/SchemaField.tsx

```diff
@@ -63,22 +65,28 @@ export const SchemaField = ({
       );
     }
     case "array": {
-      if (!propertySchema.items) {
-        throw new Error("Array schema must define items");
-      }
-
-      switch ((propertySchema.items as JSONSchema7).type) {
-        case "object": {
-          return (
-            <OptionSet label={label} name={fieldName} isDisabled={isDisabled} />
-          );
-        }
-        default: {
-          throw new Error(
-            `Unexpected propertySchema.items.type: ${propertySchema.type}`
-          );
+      if (
+        typeof propertySchema.items === "object" &&
+        !(propertySchema.items instanceof Array)
+      ) {
+        switch (propertySchema.items.type) {
+          case "object": {
+            return (
+              <OptionSet
+                label={label}
+                name={fieldName}
+                isDisabled={isDisabled}
+              />
+            );
+          }
+          default: {
+            throw new Error(
+              `Unexpected propertySchema.items.type: ${propertySchema.type}`
```

Comment:
Is this change related to the current PR? If not, I suggest removing it. 
If it is needed, please open a new issue and make the change separately.